python/mxnet/_ctypes/ndarray.py

```diff
@@ -118,6 +130,13 @@ def __init__(self, sym, flags=()):
     def __del__(self):
         check_call(_LIB.MXFreeCachedOp(self.handle))
 
+    def _is_from_np_compat_op(self, idx):
```

Comment:
Please delete this function. It was added for another design which has been abandoned.

Reply:
Done.